src/tracing/span.js

```diff
@@ -37,6 +37,14 @@ export class Span {
     return this.span.spanContext;
   }
 
+  spanId() {
```

Comment:
```suggestion
  get spanId() {
```
Can be used like a property, since it's not doing any computation, I think it can be a minor improvement.

##### non-blocking

Reply:
I agree, this is better.